Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -217,8 +218,11 @@ void PhysicalParticleContainer::InitData ()
     // Init ionization module here instead of in the PhysicalParticleContainer
     // constructor because dt is required
     if (do_field_ionization) {InitIonizationModule();}
+#ifdef PULSAR
+#else
```

Comment:
#ifndef PULSAR would be simpler here ...

Reply:
Added this now. thank you.